api/validation/schemas.py

```diff
@@ -0,0 +1,26 @@
+from marshmallow import Schema, fields, validate, INCLUDE
```

Comment:
question: since we have to cover all application endpoints with validation, is it safe to have all schemas under a single file?

Reply:
I believe it's safe if they're still "readable"/"searchable" easily
But I strongly believe in separating code in packages, so as soon as we find the schemas need better organization/structure we can do the following:
- restructure the schemas in separate files under `api/validation` package
- export them all from the `api/validation/__init__.py` so they're still all importable from `from api.validaton import CertainSchema`